spec/rails_helper.rb

```diff
@@ -8,7 +8,15 @@
 require 'capybara/rails'
 require 'sidekiq/testing'
 require 'carrierwave/test/matchers'
+require 'vcr'
+Capybara.server_port = 52662
+Capybara.javascript_driver = :webkit
```

Comment:
I would also recommend to try out poltergeist. To my experience it is much faster and a bit more stable.